src/main/java/core/basesyntax/service/FruitList.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface FruitList {
+    List<FruitTransaction> getFruitList(List<FruitTransaction> list);
```

Comment:
The method name 'getFruitList' is also vague. It should clearly describe the action it performs. If it's intended to filter or transform the list, consider a name like 'processTransactions' or 'filterTransactions'.